tests/conftest.py

```diff
@@ -1,10 +1,15 @@
 from __future__ import absolute_import, division, print_function
 
+import os
 import warnings
 
+import torch
+
 import pyro
 
 
+torch.set_default_tensor_type(os.environ.get('PYRO_TENSOR_TYPE', 'torch.DoubleTensor'))
```

Comment:
Since some of our tests don't use utility functions from `common.py` (by directly using `assert`), moving this from `common.py` to `conftest.py` which is called for each test.